src/Microsoft.Dism.Tests/DismSessionTest.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c). All rights reserved.
+//
+// Licensed under the MIT license.
+
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Dism.Tests
+{
+    public class DismSessionTest : TestBase
```

Comment:
Can you derive from `DismTestBase` instead, it calls Initialize() and Shutdown for you and provides the log to the TestOutputHelper.  Then you can remove the Initialize() and Shutdown() in `SessionOptionsBehavior`